### PR TITLE
added header for index/signup

### PR DESCRIPTION
**Pull request by Pybite**
resolved issue #12

### CODE REVIEW

File: src/global.css [hunk 1]
```diff
@@ -2,4 +2,14 @@
...
     background-color: gainsboro;
     padding: 10px;
+}
+
+.site-header {
+    background-color: hsl(194, 100%, 66%);
+    text-indent: 20px;
+}
+
+#site-header-heading {
```
Comment (nbkhope) [inline src/global.css:12]: Try to always use classes :)

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,9 @@
     <link rel="stylesheet" href="global.css">
   </head>
   <body>
+    <header class="site-header">
+      <h1><a href="index.html" id="site-header-heading">FSWD</a></h1>
```
Comment (nbkhope) [inline src/index.html:15]:
```suggestion
      <h1><a href="index.html" class="site-header-heading">FSWD</a></h1>
```

File: src/global.css (hunk 1 above)
Comment (Pybite) [reply src/global.css:12]: Ok no problem

Comment (nbkhope) [reply src/global.css]: Basically you can always start off using a class.

The case for using ID is when you absolutely cannot do without. When it's something really unique in the document. 

That you for some reason cannot target using just a class. Like a special case for targetting using JavaScript `document.getElementById`. But honestly it's hard to find that case. 

 You can think of the header as something generic that will appear in every page, so not really unique. 

You're safe always using class.

**Review by nbkhope — APPROVED** [review]

File: src/global.css (hunk 1 above)
Comment (Pybite) [reply src/global.css]: Ahh thank you for the explanation!

**Pull request merged**